Quick Wins changes to docs pages (#1922)
dbt-labs/docs.getdbt.com: website/docs/reference/resource-configs/severity.md

---
title: "Configuring test `severity`"
id: "severity"

> description: "You can use error thresholds in dbt to configure the severity of test results, or how many failed tests will produce an error or warning."

Reviewer: ```suggestion
description: "You can use error thresholds to configure the severity of test results and set when to produce an error or warning based on the number of failed tests."
```

Reviewer: sugg for present tense
